### PR TITLE
XS-3826 Updated the version numbers to accurately reflect the version…

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Updated rule versions to correct version.

### Pull request process

- [x] Full description of changes provided above.
- [x] Reviewed by another person. (+1)
- [ ] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

##### Once checklist is complete, @hefischer  @andrewperkins-wf please review for merge.

… of the rules

### CODE REVIEW

**Comment by joyreistad-wf** [comment]
+1

**Comment by sagesmith-wf** [comment]
The versions on these files seem super weird and random. Are they all versioned independently, and if so what is the purpose of that?

**Comment by allenbross-wf** [comment]
@sagesmith-wf They are versioned independently.  
"The dqc_us_rules library follows a standard semantic versioning system of MAJOR.MINOR.FIX format. Major releases are specified when a new batch of rules have been approved, coded, and accepted by the DQC after a public comment period.

The version specified by each individual rule is tied to the last release version which the rule was altered. For example, a rule being marked as v2.0.0 would have last been functionally modified during the 2.0.0 release of the DQC library."

**Comment by sagesmith-wf** [comment]
+1

File: dqc_us_rules/dqc_us_0005.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
     re.compile(r'^http://xbrl\.((sec\.gov)|(us))/dei/\d{4}-\d{2}-\d{2}')
 )
 _CODE_NAME = 'DQC.US.0005'
-_RULE_VERSION = '1.1.0'
+_RULE_VERSION = '2.1.0'
```
Comment (allenbross-wf) [inline dqc_us_rules/dqc_us_0005.py]: The rule version is correct.  I have fixed the package version number.

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by joyreistad-wf** [comment]
+1

**Comment by chrislococo-wf** [comment]
#### QA Steps: 

- Manual testing: Versions are labeled correctly
- Flake8 on dqc_us_rules: 
```
(dqc)  chrislococo [dqc] (XS-3826) $ flake8 dqc_us_rules
(dqc)  chrislococo [dqc] (XS-3826) $
```
- Flake8 on tests: 
```
(dqc)  chrislococo [dqc] (XS-3826) $ flake8 tests/
(dqc)  chrislococo [dqc] (XS-3826) $
```
- Nosetest result: 
```
(dqc)  chrislococo [dqc] (XS-3826) $ nosetests
...........................................................................................................
----------------------------------------------------------------------
Ran 107 tests in 1.392s

OK
```

#### Result: QA +10

**Pull request merged**